.github/workflows/release-semantic-automated-npm.yml

```diff
@@ -0,0 +1,70 @@
+name: Semantic Release
+on:
+  workflow_call:
+    inputs:
+      NODE_VERSION:
+        description: Node version with which the release will be executed.
+        default: 18
+        required: false
+        type: string
+
+jobs:
+  release:
+    name: Release
+    timeout-minutes: 5
+    runs-on: ubuntu-latest
+    env:
+      HAS_CONFIG: false
+    steps:
+      - name: Setup Node.js
+        uses: actions/setup-node@v3
+        with:
+          node-version: ${{ inputs.NODE_VERSION }}
+
+      - name: Install dependencies
+        run: |
+          npm i -g @semantic-release/changelog \
+            @semantic-release/git \
+            @semantic-release/npm \
+            @semantic-release/exec \
+            semantic-release
+
+      - name: Checkout
+        uses: actions/checkout@v3
+
+      - name: Check presence of release.config.js
+        run: |
+          HAS_CONFIG=$(test -f "release.config.js" && echo true || echo false)
+          echo "Configuration file release.config.js found: $HAS_CONFIG"
+          echo "HAS_CONFIG=$HAS_CONFIG" >> $GITHUB_ENV
+
+      - name: Checkout the workflow repository if release.config.js file is not provided
+        if: ${{ env.HAS_CONFIG == 'false' }}
+        uses: actions/checkout@v3
+        with:
+          repository: inpsyde/reusable-workflows
+          ref: automatic-release
+          path: workflow-repo
+
+      - name: Add release.config.js file if not provided
+        if: ${{ env.HAS_CONFIG == 'false' }}
+        run: |
+          cp ${GITHUB_WORKSPACE}/workflow-repo/templates/release-semantic-automated-npm/release.config.js .
+
+      - name: Check the presence of the configuration file
+        if: ${{ env.HAS_CONFIG == 'false' }}
+        run: |
+          if [ ! -f "release.config.js" ]; then 
+            echo "Configuration file release.config.js does not exist, aborting!" 
+            exit 1
+          fi
```

Comment:
Why do we need this check? We check for a config file in the repository of the caller workflow (https://github.com/inpsyde/reusable-workflows/pull/32/files#diff-dd37b03d2773c8f29786944f4cf0bdecf07d50ba93edb67cfa566b1df852b3fdR35-R39), and if it doesn't exist, we use the one from the reusable workflows (https://github.com/inpsyde/reusable-workflows/pull/32/files#diff-dd37b03d2773c8f29786944f4cf0bdecf07d50ba93edb67cfa566b1df852b3fdR41-R52).

IMO, the only helpful case for such an additional check would be in an E2E test, but we don't have that yet.
```suggestion
```

Reply:
Removed :+1: